### PR TITLE
[RFC] fix: setStatementQuery(String) properly called

### PR DESCRIPTION
**Pull request by typekpb**
Properly called: setStatementQuery(String), for details, see: #385

### CODE REVIEW

File: src/main/java/com/p6spy/engine/wrapper/StatementWrapper.java [hunk 1]
```diff
@@ -235,6 +243,12 @@ public int executeUpdate(String sql, String[] columnNames) throws SQLException {
 
   @Override
   public void addBatch(String sql) throws SQLException {
+    if (statementInformation.getStatementQuery() == null) {
+      statementInformation.setStatementQuery(sql);
+    } else {
+      statementInformation.setStatementQuery(sql + statementInformation.getStatementQuery());
```
Comment (felixbarny) [inline src/main/java/com/p6spy/engine/wrapper/StatementWrapper.java]: With line break maybe?

File: src/main/java/com/p6spy/engine/wrapper/StatementWrapper.java [hunk 2]
```diff
@@ -246,7 +246,7 @@ public void addBatch(String sql) throws SQLException {
     if (statementInformation.getStatementQuery() == null) {
       statementInformation.setStatementQuery(sql);
     } else {
-      statementInformation.setStatementQuery(sql + statementInformation.getStatementQuery());
+      statementInformation.setStatementQuery(sql + System.lineSeparator() + statementInformation.getStatementQuery());
```
Comment (felixbarny) [inline src/main/java/com/p6spy/engine/wrapper/StatementWrapper.java]: `System.lineSeparator()` was only introduced in Java 7, use `System.getProperty("line.separator")` instead

**Comment by typekpb** [comment]
@felixbarny can we merge this?

**Comment by felixbarny** [comment]
LGTM

**Pull request merged**